libs/genai/langchain_google_genai/chat_models.py

```diff
@@ -1006,3 +1007,14 @@ def bind_tools(
             ]
             tool_config = _tool_choice_to_tool_config(tool_choice, all_names)
         return self.bind(tools=genai_tools, tool_config=tool_config, **kwargs)
+
+    @property
+    def _supports_tool_choice(self) -> bool:
+        return "gemini-1.5" in self.model
```

Comment:
nits: there's a special enum for that:
https://github.com/langchain-ai/langchain-google/blob/e7b45726adc15e17f75f018f4542ec11aecdaeb3/libs/vertexai/langchain_google_vertexai/_utils.py#L120

Reply:
but thats in vertexai